cfn-resources/go.mod

```diff
@@ -10,7 +10,28 @@ require (
 	github.com/rs/xid v1.2.1
 	github.com/sirupsen/logrus v1.4.2
 	github.com/spf13/cast v1.3.1
-	go.mongodb.org/atlas v0.5.1-0.20201123170532-371c0d8194fe
+	go.mongodb.org/atlas v0.17.0
 	go.mongodb.org/mongo-driver v1.5.1
+)
+
+require (
+	github.com/aws/aws-lambda-go v1.13.3 // indirect
+	github.com/go-stack/stack v1.8.0 // indirect
+	github.com/golang/snappy v0.0.1 // indirect
+	github.com/google/go-querystring v1.1.0 // indirect
+	github.com/jmespath/go-jmespath v0.4.0 // indirect
+	github.com/klauspost/compress v1.9.5 // indirect
+	github.com/konsorten/go-windows-terminal-sequences v1.0.2 // indirect
+	github.com/openlyinc/pointy v1.1.2 // indirect
+	github.com/pkg/errors v0.9.1 // indirect
+	github.com/segmentio/ksuid v1.0.2 // indirect
+	github.com/xdg-go/pbkdf2 v1.0.0 // indirect
```

Comment:
Could you add some tests? How do you know that you are not introducing breaking changes?

Reply:
After library upgrade I have tested PROJECT Resource, which has the Code change.

Test: CFN SUBMIT, updated the version in AWS (PrivateExtension) and Deployed Cloud Formation Template.

Reply:
is it possible to add a github action that runs for every PR to test at least compilation? I am okay with a follow up PR

Reply:
@andreaangiolillo ,thanks for raising this. I ll raise another PR for the github action for builds, otherwise all the resources build is successful.